tests/conftest.py

```diff
@@ -27,12 +27,15 @@
     EPOCHS,
     HYPEROPT,
     INPUT_FEATURES,
+    MODEL_ECD,
+    MODEL_GBM,
     NAME,
     OUTPUT_FEATURES,
     TRAINER,
     TYPE,
 )
 from ludwig.hyperopt.run import hyperopt
+from ludwig.schema import get_schema
```

Comment:
Can you import `from ludwig.config_validation.validations import get_schema`? Want to try to remove `ludwig.schema.__init__.py`

Reply:
Done!